bentoml/yatai/web/yarn.lock

```diff
@@ -7426,8 +7426,8 @@ protobufjs@^6.8.8:
     "@protobufjs/path" "^1.1.2"
     "@protobufjs/pool" "^1.1.0"
     "@protobufjs/utf8" "^1.1.0"
-    "@types/long" "^4.0.0"
-    "@types/node" "^10.1.0"
+    "@types/long" "^4.0.1"
+    "@types/node" "^13.7.0"
```

Comment:
cc @yubozhao I'm not sure why @types/node also got updated after I run `yarn upgrade protobufjs@^6.9.0` but it seems not affecting the build nor starting up the webserver.